azuredevops/internal/service/taskagent/resource_variable_group.go

```diff
@@ -36,9 +40,28 @@ const (
 	azureKeyVaultType                         = "AzureKeyVault"
 	invalidVariableGroupIDErrorMessageFormat  = "Error parsing the variable group ID from the Terraform resource data: %v"
 	flatteningVariableGroupErrorMessageFormat = "Error flattening variable group: %v"
-	expandingVariableGroupErrorMessageFormat  = "Error expanding variable group resource data: %+v"
+	expandingVariableGroupErrorMessageFormat  = "Expanding variable group resource data: %+v"
```

Comment:
Why remove the word error here? Why remove it for this error but not the others?

Reply:
Terraform will automatically add the `Error:` prefix to each error message.
![image](https://user-images.githubusercontent.com/57888764/106857898-cc93e100-66fb-11eb-9a90-2afdad832921.png)

Reply:
Should we remove it from the other ones too then?

Reply:
I'm tend to remove it.